R/code_feedback.R

```diff
@@ -129,6 +129,8 @@
 #' @param user_code,solution_code String containing user or solution code. For
 #'   ease of use in `grade_this()`, `.user_code` or `.solution_code` are by
 #'   default retrieved from the calling environment.
+#' @param solution_code_all A list containing the code of all solutions when
```

Comment:
Link to or reference `.solution_code_all`

Reply:
Documented in [8c143b7](https://github.com/rstudio/gradethis/pull/289/commits/8c143b7b9aa0ebc66deb8059b48f02438244e62a)